consoles/sshVirtsh.pm

```diff
@@ -105,7 +105,8 @@ sub _init_xml {
     $root->appendChild($elem);
 
     $elem = $doc->createElement('description');
-    $elem->appendTextNode("openQA Instance $instance");
+    $elem->appendTextNode("openQA Instance $instance: ");
+    $elem->appendTextNode(get_required_var('NAME'));
```

Comment:
`Could not retrieve required variable NAME at ../consoles/sshVirtsh.pm line 109.` I'm such a genius